src/main/java/seedu/hirehub/logic/commands/CommentCommand.java

```diff
@@ -31,6 +31,9 @@ public class CommentCommand extends Command {
     public static final String MESSAGE_ADD_COMMENT_SUCCESS = "Added comment to Person: %1$s";
     public static final String MESSAGE_DELETE_COMMENT_SUCCESS = "Removed comment from Person: %1$s";
 
+    public static final String COMMENT_CANNOT_BE_EDITED = "Comment on candidates cannot be edited via edit method.\n"
+            + "Please use comment command instead in order to update comment on individual candidates.";
+
```

Comment:
Would it be better for this error message to be in the EditCommand class instead?

Reply:
okay sure